frontend/src/pages/integrations/octopus-deploy/create.tsx

```diff
@@ -0,0 +1,443 @@
+          }
+        },
+        sourceEnvironment: sourceEnvironment.slug,
+        secretPath
+      });
+
+      createNotification({
+        type: "success",
+        text: "Successfully created integration"
+      });
+      router.push(`/integrations/${currentWorkspace?.id}`);
+    } catch (err) {
+      createNotification({
+        type: "error",
+        text: "Failed to create integration"
+      });
+      console.error(err);
+    }
+  };
+
+  useEffect(() => {
+    if (!octopusDeployResources || !octopusDeploySpaces || !currentWorkspace) return;
+
+    reset({
+      targetResource: octopusDeployResources[0],
+      targetSpace: octopusDeploySpaces.find((space) => space.IsDefault),
+      sourceEnvironment: currentWorkspace.environments[0],
+      secretPath: "/",
+      scope: OctopusDeployScope.Project
+    });
+  }, [octopusDeploySpaces, octopusDeployResources, currentWorkspace]);
+
+  if (isProjectLoading || isLoadingOctopusDeploySpaces || isOctopusDeployResourcesLoading)
+    return (
+      <div className="flex h-full w-full items-center justify-center p-24">
+        <Spinner />
+      </div>
+    );
+
+  return (
+    <form
+      onSubmit={handleSubmit(onSubmit)}
+      className="flex h-full w-full items-center justify-center"
+    >
+      <Card className="max-w-4xl rounded-md p-8 pt-4">
+        <CardTitle className=" text-center">
+          <SiOctopusdeploy size="1.2rem" className="mr-2 mb-1 inline-block" />
+          Octopus Deploy Integration
+        </CardTitle>
+        <div className="grid grid-cols-2 gap-4">
+          <Controller
+            control={control}
+            name="sourceEnvironment"
+            render={({ field: { value, onChange }, fieldState: { error } }) => (
+              <FormControl
+                errorText={error?.message}
+                isError={Boolean(error)}
+                label="Project Environment"
+              >
+                <FilterableSelect
+                  getOptionValue={(option) => option.slug}
+                  value={value}
+                  getOptionLabel={(option) => option.name}
+                  onChange={onChange}
+                  options={currentWorkspace?.environments}
+                  placeholder="Select a project environment"
+                  isDisabled={!currentWorkspace?.environments.length}
+                />
+              </FormControl>
+            )}
+          />
+          <Controller
+            control={control}
+            name="secretPath"
+            render={({ field: { value, onChange }, fieldState: { error } }) => (
+              <FormControl isError={Boolean(error)} label="Secrets Path">
+                <Input
+                  className="mt-[1px] h-[2.46rem]"
+                  value={value}
+                  onChange={onChange}
+                  placeholder={'Provide a path (defaults to "/")'}
+                />
+              </FormControl>
+            )}
+          />
```

Comment:
let's use the component with suggestions and stuff: `<SecretPathInput />`
<img width="703" alt="image" src="https://github.com/user-attachments/assets/6f478ce6-2e23-4be4-a9cd-dca09d386a2e">